### PR TITLE
2.14.0 release

### PR DESCRIPTION
**Pull request by ealsur**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+### <a name="2.14.0"></a> 2.14.0
+
+* Add PopulateIndexMetrics to FeedOptions
+* Add backend request duration in milliseconds (BELatencyMs) to RequestDiagnosticsString
+* Improved allocations on Direct/TCP
+* Improved latency for Gateway mode users on .NET Framework by disabling Nagle algorithm
+
```
Comment (j82w) [inline changelog.md:7]: * Fixed race condition in direct + tcp mode causing SDK generated internal server errors and invalid operation exceptions 
* Fixed race condition in direct + tcp mode causing unncessary connections to be created by concurrent requests

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+### <a name="2.14.0"></a> 2.14.0
+
+* Add PopulateIndexMetrics to FeedOptions
```
Comment (j82w) [inline changelog.md:3]: @timsander1 can you please provide a one sentence explaining the feature.

**Review by kirankumarkolli — APPROVED** [review]

File: changelog.md (hunk 2 above)
Comment (timsander1) [reply changelog.md:3]: Let use: Added an option to include index metrics in the response.

Comment (j82w) [reply changelog.md:3]: Can we add more details? Add PopulateIndexMetrics to FeedOptions which allows users to get index usage during testing to check the efficiency of the query.

Comment (timsander1) [reply changelog.md:3]: Sure. We can use: Add PopulateIndexMetrics to FeedOptions which allows users to get index usage metrics during testing to improve query performance..

**Review by j82w — APPROVED** [review]

**Pull request merged**